scripts/manage/add_gae_account.sh

```diff
@@ -19,13 +19,25 @@ GAE_REQUIRED_ROLES=(storage.admin appengine.appAdmin)
 EXISTING_ROLES=$(gcloud projects get-iam-policy --filter bindings.members:$SA_EMAIL $MANAGED_PROJECT_ID \
   --flatten bindings[].members --format="value(bindings.role)")
 
+if [ "$?" != "0" ]; then
+    bold "$USER does not have permission to query IAM policy on project $MANAGED_PROJECT_ID." \
+         "Please grant the necessary permissions and re-run this command."
```

Comment:
Minor but not blocking: can we help the user by telling them what permissions need to be granted?

/cc @dorbin